### ISSUE TITLE
(Feature Request): Show RYD's like/dislike counter for videos where everything is disabled

### ISSUE THREAD

**Issue by pcroland** [Anarios/return-youtube-dislike#480]
### Extension or Userscript?

Both

### Request or suggest a new feature!

It would be nice to just show the RYD stats for videos where any like/dislike is disabled.

### Can you work on this?

- [ ] Yes
- [X] No

**Comment by cyrildtm** [comment]
This is an old topic. If the creator has disabled like and dislike then we should not see counter. Read: #437

**Comment by pcroland** [comment]
We should see it and it should show just the RYD numbers it would make much mlre sense.

**Comment by cyrildtm** [comment]
No. I'm a user. I don't want to see it. Clearly you don't represent me. 

1. I respect the video creator's decision. If they don't want to show, I don't want to see. 
2. I consider the fact that the creator hides the counts as importantly as the counter itself. If they hide it, I think of the reason. If it's really a bad video, I can figure out pretty soon and stop watching as soon as if I could see a large dislike number. 
3. Showing only plugin users counts confuses me. Normally about 0.1%-1% of all viewers will vote, and at present about 0.1%-1% of all youtube watchers use this plugin. If only plugin users votes are displayed, I will mistakenly think an unusual smaller portion of all watchers have voted, and this covers up the fact that the creator has turned off counting. Until I do the math with the view count I can't see things clearly. I don't want to mix the two signals. 

No it doesn't make sense to me.

**Comment by pcroland** [comment]
1. I don't respect the video creator's decision, just like no one respect YouTube's decision to remove dislike counts. This addon could solve both issues, YouTube being dumb and creators being dumb.
2. Showing only RYD likes/dislikes could be indicated with a little badge/icon or something, so it wouldn't confuse users.
3. Creators can circumvent RYD's ability to show dislikes if the disable likes also.

**Comment by cyrildtm** [comment]
1. Creators can circumvent RYD's ability to show dislikes if the disable
   likes also.


Disabling likes has bigger disadvantages.

I'm okay with this feature if there is an option to turn it off. I don't
want to see RYD only numbers even with an indicator icon. I see "LIKE" and
"DISLIKE" then i know there's something wrong and i can skip the video
quickly. Recognizing the indicator takes me two seconds from my life, two
seconds too long.

This reminds me. Currently the plugin cannot tell apart the reasons why
numbers are unavailable. See #472



   1. Creators can circumvent RYD's ability to show dislikes if the disable
   likes also.


You can also do beyond that. Since the plugin's user base is not that much,
you can spam likes and twist the result. Fortunately this hasn't been seen
so far, but it's not a new concept. Plausible just like ddos attack or
plugin mining. The backend has "basic" features to prevent this, but it's
not so powerful.


Anyway, I can understand why you are still using youtube even when it's
dumb. Because there's no equal competitors and the business is effectively
monopoly. But why are you still watching a particular video if the creator
is dumb?
